frame/plasm-staking/src/lib.rs

```diff
@@ -499,15 +542,26 @@ decl_event!(
     pub enum Event<T>
     where
         AccountId = <T as system::Trait>::AccountId,
+        Balance = BalanceOf<T>,
     {
         /// Validator set changed.
         NewValidators(Vec<AccountId>),
+        /// Rewards to validators. (The amount of mint tokens, The amount of burn tokens)
+        Reward(Balance, Balance),
```

Comment:
I guess is more clear to separate burned and rewarded tokens.
Burn tokens in reward event looks ironically.

Reply:
I agree. 😄